### DISCUSSION TITLE
Full screen

### DISCUSSION THREAD

**Discussion by parzival418** [mathuo/dockview#354] [Ideas]
Is there any plans for a full-screen button/mode? It would be great to be able to pop out any one window into full-screen mode to focus on a task and then be able to reduce it back to the normal view.

**Comment by mathuo** [comment]
Certainly would be a nice feature, will hopefully have some time soon to take a look into it!

**Reply by parzival418** [reply]
I was using react flex layout before this and while they supported it it never seemed to work properly for me.

**Comment by mathuo** [comment]
Implemented as of version `1.9.0`. See [here](https://dockview.dev/docs/components/dockview#maximized-groups) for details.

**Reply by husayt** [reply]
@mathuo i couldn't find a mention of fullscreen mode there. Perhaps you have an example. Thanks a lot

**Reply by mathuo** [reply]
In this context full-screen means to take one group and maximize it within the current instance, knows as Maximized Groups in the documentation. Here's a quick gif of the feature from the docs website:

![Animation](https://github.com/mathuo/dockview/assets/6710312/ef60f3d9-46f6-4a14-b7e0-b95a617465bd)
